doc/docs/deploy-manage/deploy/environment-variables.md

```diff
@@ -0,0 +1,155 @@
+| ---------------------- | ----------------- | ----------- |
+| `ETCD_SERVICE_HOST`    | N/A               | The host on which the etcd service runs. |
+| `ETCD_SERVICE_PORT`    | N/A               | The etcd port number.                    |
+| `PPS_WORKER_GRPC_PORT` | `80`              | The GRPs port number.                    |
+| `PORT`                 | `650`             | The `pachd` port number. |
+| `PPROF_PORT`            | `651`             | The port number for the `pprof` tool.     |
+| `HTTP_PORT`             | `652`             | The HTTP port number.   |
+| `PEER_PORT`             | `653`             | The port for pachd-to-pachd communication. |
+| `PPS_ETCD_PREFIX`        | `pachyderm_pps`   | ???  |
+| `NAMESPACE`            | `deafult`         | The namespace in which Pachyderm is deployed. |
+
+**pachd Configuration**
+
+| Environment Variable   | Default Value     | Description |
+| ---------------------- | ----------------- | ----------- |
+| `NUM_SHARDS`           | `32`              | The max number of `pachd` pods that can run in a single cluster. |
+| `STORAGE_BACKEND`      | `"`               | The storage backend defined for the Pachyderm cluster. By default, local storage is configured. |
+| `STORAGE_HOST_PATH`    | `"`               | The host path to storage.                    |
+| `KUBERNETES_PORT_443_TCP_ADDR` | N/A   | An IP address that Kubernetes exports automatically for your code to communicate with the Kubernetes API. Read access only. Most variables that have use the `PORT_ADDRESS_TCP_ADDR` pattern are Kubernetes environment variables. For more information, see [Kubernetes environment variables](https://kubernetes.io/docs/concepts/services-networking/service/#environment-variables). |
+| `METRICS`              | `true`         | Defines whether anonymous Pachyderm metrics are being collected or not. |
+| `BLOCK_CACHE_BYTES`    | `1G`              | The size of the block cache in `pachd`.  |
+| `WORKER_IMAGE`         | `"`               | The base Docker image that is used to run your pipeline. |
+| `WORKER_SIDECAR_IMAGE` | `"`               | The `pachd` image that is used as a worker sidecar. |
+| `WORKER_IMAGE_PULL_POLICY` | `"`           | The pull policy that defines how Docker images are pulled. The default value is `IfNotPresent`. You can set a Kubernetes image pull policy as needed. |
+| `LOG_LEVEL`          | `info`              | Verbosity of the log output. If you want to disable logging, set this variable to `0`. |
+| `IAM_ROLE`           | `"`                 | The role that defines permissions for Pachyderm in AWS. |
+| `IMAGE_PULL_SECRET`  | `"`                 | The Kubernetes secret for image pull credentials. |
+| `NO_EXPOSE_DOCKER_SOCKET` | `false`        | Controls whether you can build images using the `--build` command. |
+| `EXPOSE_OBJECT_API`  | `false`             | Controls access to internal Pachyderm API. |
+| `PACHD_MEMORY_REQUEST` | `1T`              | ??? |
```

Comment:
still ??? here

Reply:
Hey @msteffen, any suggestions on the `PACHD_MEMORY_REQUEST` variable?

Reply:
Removing this for now as we need to merge this today and will address later if needed.